Packs/PrismaSASE/Integrations/PrismaSASE/PrismaSASE.yml

```
@@ -0,0 +1,779 @@
+category: Network Security
+commonfields:
+  id: Prisma SASE
+  version: -1
+configuration:
+- defaultvalue: https://api.sase.paloaltonetworks.com
+  display: API Gateway URL
+  name: url
+  required: true
+  type: 0
+- display: API Client ID
+  displaypassword: API Client Secret
+  name: credentials
+  required: true
+  type: 9
+- defaultvalue: "false"
+  display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Default Tenant Services Group ID to use for API calls.
+  name: tsg_id
+  required: false
+  type: 0
```

Maybe better this way - what do you think? 
```suggestion
- display: Tenant Services Group ID
  name: tsg_id
  required: false
  type: 0
  additionalinfo: Default Tenant Services Group ID to use for API calls.
```

---

updated.